src/commands/functions/create.js

```diff
@@ -197,22 +197,32 @@ const pickTemplate = async function () {
 const DEFAULT_PRIORITY = 999
 
 /* get functions dir (and make it if necessary) */
-const ensureFunctionDirExists = function (context, flags, config) {
-  const functionsDir = config.build && config.build.functions
-  if (!functionsDir) {
-    context.log(`${NETLIFYDEVLOG} No functions folder specified in netlify.toml`)
-    process.exit(1)
+const ensureFunctionDirExists = async function (context, flags, config) {
+  let functionsDirHolder = config.build && config.build.functions
+  if (!functionsDirHolder) {
+    context.log(`${NETLIFYDEVLOG} functions folder not specified in netlify.toml or UI settings`)
+    const { functionsDir } = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'functionsDir',
+        message: 'Enter the path where your functions should live: netlify/functions:',
+        default: 'netlify/functions',
+      },
+    ])
+
+    functionsDirHolder = functionsDir
   }
-  if (!fs.existsSync(functionsDir)) {
+
+  if (!fs.existsSync(functionsDirHolder)) {
     context.log(
       `${NETLIFYDEVLOG} functions folder ${chalk.magenta.inverse(
-        functionsDir,
+        functionsDirHolder,
       )} specified in netlify.toml but folder not found, creating it...`,
```

Comment:
It seems that if the functions directory is not configured we first print `functions folder not specified in netlify.toml` and later on print `<functions-folder> specified in netlify.toml but folder not found`.
Should we change the latter message in that case?

Reply:
Yeah cc @klavavej, I think we can put `Creating functions folder `${functionsDirHolder}...`